src/System.Memory/tests/System.Memory.Tests.csproj

```diff
@@ -0,0 +1,43 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>$(NetFrameworkMinimum)</TargetFramework>
+    <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
+    <!-- Don't run .NET Framework only tests on non Windows. -->
+    <SkipTests Condition="'$(OS)' != 'Windows_NT'">true</SkipTests>
+    <IsPartialFacadeAssembly Condition="$([MSBuild]::GetTargetFrameworkIdentifier('$(TargetFramework)')) == '.NETCoreApp'">true</IsPartialFacadeAssembly>
```

Comment:
This line doesn't make sense in a test project, which never is a facade assembly. I suggest to remove it and replace the conditions below directly with TFI conditions.

Reply:
Fixed.